### PR TITLE
[proofs] Alethe: fixing formatting and adding missing comment

### PR DESCRIPTION
**Pull request by HanielB**

### CODE REVIEW

File: src/proof/alethe/alethe_post_processor.cpp [hunk 1]
```diff
@@ -204,7 +204,7 @@ bool AletheProofPostprocessCallback::update(Node res,
     // VP8: (cl (=> (and F1 ... Fn))) (cl (not (=> (and F1 ... Fn) false))
```
Comment (Lachnitt) [inline src/proof/alethe/alethe_post_processor.cpp:204]: This should be VP8: (cl (not (=> (and F1 ... Fn) false))

Comment (HanielB) [reply src/proof/alethe/alethe_post_processor.cpp:204]: Actually I think it's `// VP8: (cl (=> (and F1 ... Fn) false))`?

Comment (Lachnitt) [reply src/proof/alethe/alethe_post_processor.cpp:204]: Ah yes of course!

**Comment by Lachnitt** [comment]
LGTM

**Review by ajreynol — APPROVED** [review]

**Pull request merged**